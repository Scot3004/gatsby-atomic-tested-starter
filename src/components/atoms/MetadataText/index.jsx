import styled from 'styled-components';

export const MetadataText = styled.span`
  margin: 0 1rem 0 0.2rem;
  font-weight: lighter;
`;

export default MetadataText;
