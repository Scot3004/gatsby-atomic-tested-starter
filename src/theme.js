const theme = {
  backgroundColor: '#f5f5f5',
  textColor: '#000',
  mutedColor: '#555',
  primaryColor: '#663399',
  visitedColor: '#402060'
};

export default theme;
