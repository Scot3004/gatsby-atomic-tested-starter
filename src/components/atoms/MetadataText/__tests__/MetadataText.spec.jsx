import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import theme from '../../../../theme';
import MetadataText from '..';

describe('When I see the metadata of a post', () => {
  const tree = renderer.create(<MetadataText theme={theme} />).toJSON();
  test('Then the text should be lighther', () => {
    expect(tree).toMatchSnapshot();
  });
});
