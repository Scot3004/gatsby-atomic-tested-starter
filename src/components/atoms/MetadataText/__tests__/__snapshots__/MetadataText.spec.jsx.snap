// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When I see the metadata of a post Then the text should be lighther 1`] = `
.c0 {
  margin: 0 1rem 0 0.2rem;
  font-weight: lighter;
}

<span
  className="c0"
/>
`;
